README.md

```diff
@@ -124,7 +124,7 @@ You can make the default user a site admin by adding the following to `applicati
 ```
 simple-report:
   site-admin-emails:
-    - bob@sample.com
+    - bob@example.com
```

Comment:
I'm fine with this change, but: why?

Reply:
this email slowly drifted away from the other @example.com addresses used for mock users, by accident. this realigns it with the others.

Reply:
FYI. `example` is a reserved TLD name and is meant to be used in places like this. https://tools.ietf.org/html/rfc2606

Reply:
uh oh, apologies to Bob and Jamar at Sample.com if they received any Okta emails from me then!